### DISCUSSION TITLE
How to use Volar 2.0

### DISCUSSION THREAD

**Discussion by BracketJohn** [vuejs/language-tools#3899] [Q&A] [answered]
Hey there👋 

Since Volar 2.0 was released recently, it would be great to use vue language tools with it.

There is a thread here that includes some alpha 2.0 vs code extensions, but this was not updated since December.

Is there a more recent way to use Volar 2.0 or is the alpha vs code plugin approach still the most recent one?

Thanks in advance and have a good day (:

**Comment by so1ve** [comment, marked as answer]
You can clone this repository, `cd extensions/vscode`, `npm run pack`.

**Reply by BracketJohn** [reply]
Cool, thank you very much 🙌

**Reply by BracketJohn** [reply]
Quick followup: How do I then install the resulting `volar-1.8.27.tgz` to vscode? Do I:
1. unpack `volar-1.8.27.tgz` to get the `package/` folder
2. `mv package ~/.vscode/extensions/vue-language-tools-2-test`

This is what I tested based on https://vscode-docs.readthedocs.io/en/stable/extensions/install-extension/ but could not get to work.

Thanks already for helping out here & investing your time.
